src/nnet3/nnet-simple-component.cc

```diff
@@ -154,6 +186,8 @@ void DropoutComponent::Read(std::istream &is, bool binary) {
   ReadBasicType(is, binary, &dim_);
   ExpectToken(is, binary, "<DropoutProportion>");
   ReadBasicType(is, binary, &dropout_proportion_);
+  ExpectToken(is, binary, "<DropoutPerFrame>");
```

Comment:
Make this backcompatible. Change this to ReadToken and then add an if condition to check which token is present.
See other components where ReadToken is used.